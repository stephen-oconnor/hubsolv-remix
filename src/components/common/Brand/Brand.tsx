import React from 'react';
import IconLogo from '../../Icons/IconLogo';
import s from './styles.module.scss';

const Brand: React.SFC = () => {
	return (
		<div className={s.brand}>
			<IconLogo width={40} height={40} />
			<h1 className={s.name}>ReMix</h1>
		</div>
	);
};

export default Brand;
