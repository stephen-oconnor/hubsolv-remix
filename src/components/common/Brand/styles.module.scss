@import '../../../styles//base/config.scss';

.brand {
	display: flex;
	align-items: center;
}

.name {
	font-size: 2.4rem;
	font-weight: $font_weight_normal;
	color: $text_colour_light;
	margin: 0 0 0 1.2rem;
}
