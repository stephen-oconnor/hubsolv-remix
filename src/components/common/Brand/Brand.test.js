import React from 'react';
import Brand from './Brand';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

describe('<Brand />', () => {
	it('matches the snapshot', () => {
		const tree = shallow(<Brand />);
		expect(toJson(tree)).toMatchSnapshot();
	});

	it('should contain 1 h1 element', () => {
		const wrapper = shallow(<Brand />);
		expect(wrapper.find('h1').length).toBe(1);
	});

	it('should have an h1 with correct text', () => {
		const wrapper = shallow(<Brand />);
		expect(wrapper.find('h1').text()).toBe('ReMix');
	});
});
