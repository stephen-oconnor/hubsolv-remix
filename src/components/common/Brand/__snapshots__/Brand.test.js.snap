// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Brand /> matches the snapshot 1`] = `
<div
  className="brand"
>
  <IconLogo
    height={40}
    width={40}
  />
  <h1
    className="name"
  >
    ReMix
  </h1>
</div>
`;
